pubspec.yaml

```diff
@@ -17,13 +17,36 @@ dependencies:
   cupertino_icons: ^1.0.2
   flutter:
     sdk: flutter
+  flutter_launcher_icons: ^0.13.1
   solid:
-    path: solid # Use a local path whilst solid is under development.
-    ## git:
-    ##   url: https://github.com/anusii/solid
-    ##   ref: main
+    path: ../solid # Use a local path whilst solid is under development.
+    # git:
+    #   url: https://github.com/anusii/solid
+    #   ref: main
   universal_io: ^2.2.2
   window_manager: ^0.3.7
+  
+
+# Define laucher icons for all platforms
```

Comment:
`laucher` needs an `n` and please add a blank line after the comment. Thanks.

Reply:
Resolved